src/main/java/sendinblue/Json.java

```diff
@@ -10,356 +10,356 @@
  * Do not edit the class manually.
  */
 
```

Comment:
It's mentioned in the document that - `Do not change anything in file- src/main/java/sendinblue/Json.java`
Is this intentional?